package.json

```diff
@@ -87,6 +87,7 @@
     }
   },
   "dependencies": {
+    "@purple/phoenix-components": "^2.5.1",
```

Comment:
why add this as a dependency?

Reply:
Thanks for asking Alex. I've noticed this being added after I did the "npm i @purple/phoenix-components". I've asked Aleš who said I shouldn't worry about it, but it doesn't seem right to me to modify the package.json in this repo. Maybe you can discard this change from the request?

Reply:
yeah that's funny. I'm not sure how it added it, but it could definitely lead to some funky behavior when installing the package